### PR TITLE
Publication building blocks

### PR DESCRIPTION
**Pull request by mturilli**

### CODE REVIEW

File: _publications/turilli2018building [hunk 1]
```diff
@@ -0,0 +1,14 @@
+M. Turilli, A. Merzky, V. Balasubramanian, S. Jha. “Building Blocks for Workflow Sys-tem Middleware”. Accepted in Proceedings of the 18th IEEE/ACM International Sym-posium on Cluster, Cloud and Grid Computing (CCGrid), 2018.
```
Comment (iparask) [inline _publications/turilli2018building]: This line blocks the file from being rendered. The header should be the first thing. I am not sure if you want it or not.

File: _publications/turilli2018building [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+title: "Building Blocks for Workflow System Middleware"
+collection: publications
+permalink: /publications/turilli2018building
+type: pub
+date: 2018-07-16
+author: "M. Turilli, A. Merzky, V. Balasubramanian, S. Jha"
+venue: "Proceedings of the 18th IEEE/ACM International Symposium on Cluster, Cloud and Grid Computing (CCGrid)"
+paperurl: https://ieeexplore.ieee.org/document/8411040
+---
+
```
Comment (iparask) [inline _publications/turilli2018building]: Don't you want to copy and paste the abstract here?

**Review by iparask — CHANGES_REQUESTED** [review]
Also the file is missing its extension. can you add `.md`?

File: _publications/turilli2018building (hunk 1 above)
Comment (mturilli) [reply _publications/turilli2018building]: Apologies, it was a left over from a C&P

File: _publications/turilli2018building (hunk 2 above)
Comment (mturilli) [reply _publications/turilli2018building]: Done

**Review by iparask — APPROVED** [review]

**Pull request merged**